types/assertrc/index.d.ts

```diff
@@ -0,0 +1,22 @@
+// Type definitions for assertrc 1.0
+// Project: https://github.com/Soldy/assertrc
+// Definitions by: Soldy <https://github.com/Soldy>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+export class assertBase {
```

Comment:
That class shouldn't be exported, otherwise the type declaration indicates we can do something like this:
```ts
import { assertBase, base } from "assertrc";
const c = new assertBase();
```
which doesn't work at run time.